ATM33xx-5/drivers/dma/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2024 Atmosic
+#
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_include_directories(.)
+zephyr_sources_ifdef(CONFIG_ATM_DMA dma.c)
```

Comment:
For the ATM33 platform, this implementation is provided by the WSTK.  It should not be duplicated in this manner.